src/components/PayPal/Paypal.tsx

```diff
@@ -25,6 +25,20 @@ class PaypalElement extends UIElement<PayPalElementProps> {
         this.handleSubmit = this.handleSubmit.bind(this);
     }
 
+    // Required for transition period (until configuration object becomes the norm)
+    // - if merchant has defined properties directly in props, use these instead
+    protected formatProps(props) {
+        const { configuration, intent, merchantId } = props;
+        return {
+            ...props,
+            configuration: {
+                ...configuration,
+                ...(intent && { intent }),
+                ...(merchantId && { merchantId })
```

Comment:
Shall we do a similar thing for cards? Since `koreanAuthenticationRequired` can be both inside and outside the `configuration` object.

Reply:
I already do:
```
configuration: {
                ...props.configuration,
                ...(props.koreanAuthenticationRequired !== undefined && { koreanAuthenticationRequired: props.koreanAuthenticationRequired })
            }
```